elixir/apps/domain/lib/domain/crypto.ex

```diff
@@ -3,7 +3,6 @@ defmodule Domain.Crypto do
 
   def psk do
     random_token(@wg_psk_length, encoder: :base64)
-    |> String.slice(0, @wg_psk_length)
```

Comment:
@AndrewDryga I had to remove this otherwise the string isn't valid base64 and we can't decode it. If we are receiving this as a string, the encoding needs to be well-defined.